src/Reflection/ReflectionProperty.php

```diff
@@ -213,14 +233,23 @@ public function isDefault(): bool
         return $this->declaredAtCompileTime;
     }
 
+    public function isDynamic(): bool
+    {
+        return ! $this->isDefault();
+    }
+
     /**
      * Get the core-reflection-compatible modifier values.
      *
      * @return int-mask-of<ReflectionPropertyAdapter::IS_*>
      */
     public function getModifiers(): int
     {
-        return $this->modifiers;
+        /** @var int-mask-of<ReflectionPropertyAdapter::IS_*> $modifiers */
```

Comment:
This is kinda weird - is our tooling not able to determine this sum's result?  :thinking:

Reply:
Yes, PHPStan needs it currently.